examples/basic-infection/people.rs

```diff
@@ -0,0 +1,115 @@
+use ixa::context::Context;
+use ixa::define_data_plugin;
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Hash, Eq, PartialEq, Clone, Copy, Serialize, Deserialize)]
+pub enum InfectionStatus {
+    S,
+    I,
+    R,
+}
+
+#[derive(Copy, Clone)]
+pub struct InfectionStatusEvent {
+    pub _prev_status: InfectionStatus,
```

Comment:
Why underscore?

Reply:
I can't get around pre-commit if I don't name it this way.

Reply:
Okay, found a way to do this :)